_posts/2015-09-04-channel.html

```diff
@@ -5822,5 +5822,66 @@
 
 		<p>Your receiver should verify the signature provided in the request <strong>X-Hook-Signature</strong> header, using <strong>HMAC-SHA1</strong> algorithm with the the previously exchanged secret and the full request body as-is.</p>
 	</section>
+	<section>
+		<h2>Encoder settings</h2>
+		<p>
+			If you wish to use an non-IBM Cloud Video provided broadcaster application or appliance (like Flash Live Media Encoder,
+			Wirecast, Tricaster, etc., you will need a stream key or channel key to authenticate with IBM Cloud Video and proving
+			that broadcasting to that particular channel is allowed. Also you should get the ingest point where your device or
+			application should connect.<br>
+		</p>
+		<h4>Getting encoder settings</h4>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">GET</td>
+				<td><a>https://api.ustream.tv/channels/CHANNEL_ID/encoder-settings.json</a></td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>On success a response with HTTP status 200 OK is returned.</p>
+
+		<table class="table responsive">
+			<tr>
+				<th>key</th>
+				<th>type</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>rtmp_urls</td>
+				<td>array of string</td>
+				<td>List of RTMP ingest urls to the channels</td>
+			</tr>
+			<tr>
+				<td>channel_key</td>
+				<td>string</td>
+				<td>Permanent authentication key to channel to broadcasting</td>
+			</tr>
+			<tr>
+				<td>streaming_key</td>
+				<td>string</td>
+				<td>Temporary authentication key to channel to broadcasting, it will expire after 1 hour. If you request settings again within 1 hour the original key will extend, after one hour new key will be generated.</td>
```

Comment:
Legyen szerintem "Permanent authentication key for broadcasting on the specified channel which will expire after 1 hour ..."